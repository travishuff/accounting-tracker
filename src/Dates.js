import React from 'react'
import PropTypes from 'prop-types'

import { Form } from 'semantic-ui-react'

const Dates = ({ end, handleDateChange, start }) => {
  return (
    <Form>
      <Form.Group>
        <Form.Input
          label="Start Date:"
          width={4}
          type="date"
          name="start"
          value={start}
          onChange={handleDateChange}
          onBlur={handleDateChange}
        />
        <Form.Input
          label="End Date:"
          width={4}
          type="date"
          name="end"
          value={end}
          onChange={handleDateChange}
          onBlur={handleDateChange}
        />
      </Form.Group>
    </Form>
  )
}

Dates.propTypes = {
  end: PropTypes.string,
  handleDateChange: PropTypes.func,
  start: PropTypes.string,
}

export default Dates
