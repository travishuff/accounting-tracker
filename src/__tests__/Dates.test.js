import React from 'react'
import { create } from 'react-test-renderer'

import Dates from '../Dates'

// Snapshot test
test('snapshot', () => {
  const c = create(<Dates />)

  expect(c.toJSON()).toMatchSnapshot()
})
