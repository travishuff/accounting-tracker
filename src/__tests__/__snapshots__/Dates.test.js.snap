// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot 1`] = `
<form
  className="ui form"
  onSubmit={[Function]}
>
  <div
    className="fields"
  >
    <div
      className="four wide field"
    >
      <label>
        Start Date:
      </label>
      <div
        className="ui input"
      >
        <input
          name="start"
          onChange={[Function]}
          type="date"
        />
      </div>
    </div>
    <div
      className="four wide field"
    >
      <label>
        End Date:
      </label>
      <div
        className="ui input"
      >
        <input
          name="end"
          onChange={[Function]}
          type="date"
        />
      </div>
    </div>
  </div>
</form>
`;
